scripts/zones/Mine_Shaft_2716/npcs/Armoury_Crate.lua

```diff
@@ -108,7 +108,7 @@ local loot =
         },
         {
             { itemid = 0,     droprate = 900 }, -- Nothing
-            { itemid = 14799, droprate = 100 }, -- Feyuh's Earring
+            { itemid = 14800, droprate = 100 }, -- Feyuh's Earring
```

Comment:
Instead of using the hard-coded ID, we should add the item name and ID to the `scipts/enum/items.lua` and then use `xi.item.NAME` instead.